dashboard/src/app/api/metrics/route.ts

```diff
@@ -0,0 +1,31 @@
+import { NextRequest } from 'next/server';
+import { collectDefaultMetrics, register } from 'prom-client';
+import { env } from '@/lib/env';
+
+let metricsInitialized = false;
+
+// Required to avoid issues with hot reloading registering metrics multiple times
+if (env.ENABLE_MONITORING && !metricsInitialized) {
+  collectDefaultMetrics({ register });
+  metricsInitialized = true;
+}
```

Comment:
Consider hiding this behind some form of flag, since fixing hot reloading does not feel right to be running on production. Thoughts?